print_docs.py

```diff
@@ -99,6 +99,27 @@ def library_link(filename, line=None):
            else mathlib_github_src_root + filename.split('mathlib/src/', 1)[1]
   return root + ('#L' + str(line) if line is not None else '')
 
+def name_in_decl(decl_name, dmap):
+  if dmap['name'] == decl_name:
+    return True
+  if decl_name in [sf[0] for sf in dmap['structure_fields']]:
+    return True
+  if decl_name in [sf[0] for sf in dmap['constructors']]:
+    return True
+  return False
+    
+
+def library_link_from_decl_name(decl_name, decl_loc, file_map):
+  try:
+    e = next(d for d in file_map[decl_loc] if name_in_decl(decl_name, d))
+  except StopIteration:
+    if decl_name[-3:] == '.mk':
+      return library_link_from_decl_name(decl_name[:-3], decl_loc, file_map)
+    print(decl_name)
```

Comment:
This is debugging code, right?

Reply:
Indeed. I've left it in, but clarified the debug output.